src/Controllers/UserController.php

```diff
@@ -35,4 +38,19 @@ public function show( Request $request, Response $response, $id ) {
 			'user' => $user
 		] );
 	}
+
+	public function login( Request $request, Response $response){
+	
+		$require_login = get_option('require_login');
```

Comment:
@Perumal605 , please change the option name to "dt_launcher_require_login" to make sure it doesn't conflict with any other options set by the theme or plugins.